website/pages/docs/secrets/databases/index.mdx

```diff
@@ -13,7 +13,7 @@ description: |-
 
 The database secrets engine generates database credentials dynamically based on
 configured roles. It works with a number of different databases through a plugin
-interface. There are a number of builtin database types and an exposed framework
+interface. There are a number of builtin database types, and an exposed framework
```

Comment:
Perhaps `builtin` -> `built in` to match what's in the rest of the docs? Or vice versa.

Reply:
:+1: